internal/dnfjson/cache_test.go

```diff
@@ -103,24 +103,24 @@ var testCfgs = map[string]testCache{
 		"df2665154150abf76f4d86156228a75c39f3f31a79d4a861d76b1edd89814b62-98177081b9162766/repodata/repomd.xml":                                                                                      fileInfo{12817, 300},
 	},
 	"fake-real": { // fake but resembling real data
-		"1111111111111111111111111111111111111111111111111111111111111111.solv":           fileInfo{100, 0},
```

Comment:
Unfortunately golang's os.ReadDir() returns entries in alphabetical order and since for our test data, entries are sorted both alphabetically and by mtime, tests pass even if no sorting is done at all.
So this change breaks the alphabetical order of data.

Reply:
Good thinking.  Though I think it doesn't matter here because maps are unordered.  Did this have any effect on the result of the test?
I wouldn't revert it, keep the change, I'd be curious to know though.

Reply:
This is only about `rpmCache.reporecency` since it's an ordered array, but not any other fields.
Since the the test data files are written to disk then read using `os.ReadDir(r.root)`, they come out in the right order automatically.